substrate/frame/revive/src/tests/revm_test_utils.rs

```diff
@@ -0,0 +1,27 @@
+
+use revm::bytecode::opcode::*;
+
+pub fn make_evm_bytecode_from_runtime_code(runtime_code: &Vec<u8>) -> Vec<u8> {
+    let runtime_code_len = runtime_code.len();
+    assert!(runtime_code_len < 256);
+    let mut init_code: Vec<u8> = vec![
+        vec![PUSH1, 0x80_u8],
+        vec![PUSH1, 0x40_u8],
+        vec![MSTORE],
+        vec![PUSH1, 0x40_u8],
+        vec![MLOAD],
+        vec![PUSH1, runtime_code_len as u8],
+        vec![PUSH1, 0x13_u8],
+        vec![DUP3],
+        vec![CODECOPY],
+        vec![PUSH1, runtime_code_len as u8],
+        vec![SWAP1],
+        vec![RETURN],
+        vec![INVALID],
+    ]
+    .into_iter()
+    .flatten()
+    .collect();
```

Comment:
can we keep all these bytecode example minimal and well documented 
e.g here

```suggestion
	let mut init_code: Vec<u8> = vec![
		// Copy runtime code from init code to memory
		vec![PUSH1, runtime_code_len as u8], // size of runtime code to copy
		vec![PUSH1, 11 as u8],               // Offset in init code where runtime code starts
		vec![PUSH0],                         // Destination in memory (0x00)
		vec![CODECOPY],                      // Copy runtime code to memory
		// Return the runtime code
		vec![PUSH1, runtime_code_len as u8], // Size of runtime code to return
		vec![PUSH1, 0x00],                   // Offset in memory where runtime code was copied
		vec![RETURN],                        // Return the runtime code
	]
	.into_iter()
	.flatten()
	.collect();

```

Reply:
also let's move that to sol.rs to avoid creating an extra file here